src/skin/popup.css

```diff
@@ -332,6 +340,109 @@ font-size: 16px;
   margin: 5px;}
 #report_close:hover {
   text-decoration: underline;}
+#share_overlay {
```

Comment:
This is the same CSS as for `#overlay`, right? We should refactor to avoid redundancy ... apply to a common class instead?